Editor/MenuActions/Geometry/OffsetElements.cs

```diff
@@ -59,6 +59,41 @@ protected override MenuActionState optionsMenuState
             get { return MenuActionState.VisibleAndEnabled; }
         }
 
+#if UNITY_2023_2_OR_NEWER
+        protected internal override VisualElement CreateSettingsContent()
+        {
+            var root = new VisualElement();
+
+            var dist = s_Translation.value;
+            var coord = s_CoordinateSpace.value;
+
+            var spaceField = new EnumField("Coordinate Space", coord);
+            root.Add(spaceField);
+            spaceField.RegisterCallback<ChangeEvent<string>>(evt =>
+            {
+                Enum.TryParse(evt.newValue, out CoordinateSpace newValue);
+                if (s_CoordinateSpace.value == newValue)
+                    return;
+                s_CoordinateSpace.SetValue(newValue);
+                ProBuilderSettings.Save();
+            });
+
+            var distField = new Vector3Field("Translate");
+            distField.SetValueWithoutNotify(dist);
+            root.Add(distField);
+            distField.RegisterCallback<ChangeEvent<Vector3>>(evt =>
+            {
+                s_Translation.SetValue(evt.newValue, true);
```

Comment:
missing `Save` here.

also, is there not any setter that handles the save automatically? or for that matter, aren't settings saved on application exit or domain unload?

Reply:
I changed that in all the modified files to use SetValue that saves the preferences rather that value = ... + ProBuilder.Save

Reply:
[328b20c](https://github.com/Unity-Technologies/com.unity.probuilder/pull/468/commits/328b20c731c9577a69e3d58270ecdd0aa1f92a12)